docs/src/tut/docs.md

```diff
@@ -0,0 +1,149 @@
+---
+layout: docs
+---
+
+# Get started
+
+The motivation behind this library is to provide a simple, easy way to provide generated date and time instances that are useful to your own domain.
+
+This library is relatively immature. Any issues, suggestions or criticisms are more than welcome.
+
+For SBT, you can add the dependency to your project's build file:
+
+```scala
+"com.fortysevendeg" %% "scalacheck-datetime" % "0.1-SNAPSHOT"
```

Comment:
Hadn't thought about that as it's not published yet. Will make sure the docs are accurate when it's published.